### DISCUSSION TITLE
Problema calcolo ritardo/uscite

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#373] [Richieste di aiuto]
Mettendo il ritardo di 30 minuti a un alunno, ho visto che il registro non lo calcola per il conteggio , infatti dice "numero di ritardo 1 di cui 0 considerato per il conteggio del numero massimo ammesso" nonostante in parametri sia stato inserito un obbligo di soli 10 minuti , sforati i quali bisogna presentare la giustifica,poi io cambierei anche il layout delle assenze ,lato genitore/alunni .Metterei una semplice griglia con la leggenda A=assente,R=ritardo,U=uscita anticipata ,AG=assenza giustificata etc,senza tutte quelle cose ,conteggio giorni etc

**Comment by trinko** [comment]
La gestione è corretta. Leggi:
 [Gestione ritardi e uscite anticipate](https://github.com/iisgiua/giuaschool/discussions/376)
